packages/material-ui/src/Select/SelectInput.js

```diff
@@ -48,7 +48,6 @@ const SelectInput = React.forwardRef(function SelectInput(props, ref) {
     required,
     SelectDisplayProps,
     tabIndex: tabIndexProp,
-    type = 'hidden',
```

Comment:
The InputBase component set type to text, the Select component set type to undefined. It shouldn't work as expected:

![Capture d’écran 2019-09-13 à 11 31 20](https://user-images.githubusercontent.com/3165635/64852684-1e752c00-d61a-11e9-9066-36ec0f64ba58.png)

Reply:
And that's why Input has no place in the Select.

Reply:
So far, we have used the Input so people can customize the two at the same time: https://material-ui.com/components/selects/#customized-selects.